packages/react-sdk/docusaurus/docs/React/03-ui-components/core/participant-view.mdx

```diff
@@ -3,8 +3,130 @@ id: participant-view
 title: Participant
 ---
 
-:::warning
+The `ParticipantView` component is a core component that renders a participant's video and plays the participant's audio.
 
-TODO: write about: https://github.com/GetStream/stream-video-js/tree/main/packages/react-sdk/src/core/components/ParticipantView
+It can toggle between video and avatar based on the participant's video state. It displays information about the participant, their device settings and connection quality. It also provides action buttons (for example, to pin the participant).
+
+To optimize performance,
+
+- the component won't stream video for participants that are not visible in the viewport
+- for visible participants it will request the appropriate video track quality and resolution (dynascale)
+
+![Preview of the Call controls component.](../../assets/03-ui-components/participant.png)
+
+:::note
+
+Please note that we suggest not to create a custom `ParticipantView` component as it contains complex, low-level logic related to performance. Please refer to the [Customization section](#customization) to see how you can tailor the component to your needs.
 
 :::
+
+## General usage
```

Comment:
I would mention somewhere in the document that the component is used in multiple call layout components + link to the call-layout.mdx doc.

Reply:
Done